tests/ecmascript/test-bi-math-cbrt.js

```diff
@@ -0,0 +1,47 @@
+/*
+ *  Math.cbrt()
+ */
+
+/*@include util-number.js@*/
+
+/*===
+function true false true
+1
+0
+-0
+Infinity
+-Infinity
+NaN
+2
+4
+10
+-2
+2.5
+-2.5
+812
+===*/
+
+function test() {
+    var pd = Object.getOwnPropertyDescriptor(Math, 'cbrt');
+    print(typeof pd.value, pd.writable, pd.enumerable, pd.configurable);
+    print(Math.cbrt.length);
+
+    printExact(Math.cbrt(0));
+    printExact(Math.cbrt(-0));
+    printExact(Math.cbrt(Infinity));
```

Comment:
Hm, I guess that makes sense for purity since Duktape allows you to remove bindings at compile time.  I wrote the above under the assumption that the global constants would always be available (they are non-writable and non-configurable, so this is a reasonable assumption in most environments), but for Duktape that may not always hold.